### ISSUE TITLE
Can't get the profiler screenshot feature working in the React DevTools

### ISSUE THREAD

**Issue by gbgdev** [facebook/react#17085]
I've recently come across [this tweet](https://twitter.com/brian_d_vaughn/status/1113200027835310080) and saw that the new DevTools profiler apparently would be able to capture images of the DOM after each commit and display them on the right side (below the commit information). 

Was this feature indeed added to DevTools 4.2.0? Or is it coming out in a future version?

Just asking because I didn't manage to get it working while trying with my projects.

**React**: 16.10.2
**Browser:** Google Chrome 77.0.3865.90
**DevTools:** 4.2.0-a8b8ffb89

**Comment by bvaughn** [comment]
This feature was not released in the final version because the performance wasn't very good with the current screenshot API. I tried [throttling](https://github.com/bvaughn/react-devtools-experimental/commit/d766d3b2e554e2998fbc3b30c6585520c75d95f3) but that just resulted in an awkward user experience, since only some commits would have screenshots. So [I removed it entirely](https://github.com/bvaughn/react-devtools-experimental/commit/271aa75cf2685ff62932863c1b1eaf08b1898d1a).
